rabbitmq/client.go

```diff
@@ -0,0 +1,109 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/palantir/stacktrace"
+
+	"github.com/streadway/amqp"
+)
+
+// ReconnectDelay ...
+const ReconnectDelay = 2
+const RetryAttempts = 5
+
+type RabbitMqClientConfig struct {
+	Host                  string
+	Port                  int
+	Username              string
+	Password              string
```

Comment:
Why do we need all these `Host`, `Port`, `Username` and `Password`?

Imo, it's better to pass a connection URI cause we don't have to handle cases where no `Port` is given, but you add a `:` after `Host` (which is currently the case and will break).